tests/integration/generate/test_convenience.py

```diff
@@ -14,6 +14,11 @@ def test_string():
 
 
 @assert_report_generated
+def test_unicode_string():
+    return Block(u"Hello World!", title=u"Salutations")
```

Comment:
I suggest you test it with some non-ASCII characters, to make sure you're not converting it to ASCII somewhere by accident. E.g. `Block(u"£")`.